data/hai/midas.txt

```diff
@@ -0,0 +1,1010 @@
+		system
+			distance 2 5
+		fleet "Marauders" 3
+	to fail
+		or
+			not "outfit: Gold Bars"
+			has "Midas: Betrayal For The Bars: offered"
+	to complete
+		never
+
+
+
+
+phrase "midas defeated"
+	word
+		`After disabling the last of the pirate ships, you wonder what could have motivated such an attack. Do you possess something valuable to them? When they entered the system, they only sent one message: "nemo me impune lacessit". It is probably best to store whatever they were interested in on a planet (using U) or to get rid of it.`
+
+# Attacks to get the bars back, in Hai and human space; as they have more influence in human space they are more numerous there
+mission "Midas: Nemo Me Impune Lacessit"
+	minor
+	landing
+	invisible
+	repeat
+	source
+		near "Ya Hai" 10
+	to offer
+		has "outfit: Gold Bars"
+		not "midas: sold bars"
+		not "midas: proven worthy"
+		random < 3
+	npc disable
+		government "Midas"
+		personality plunders disables heroic nemesis marked
+		system
+			distance 2 5
+		fleet "Marauders" 3
+		dialog phrase "midas defeated"
+	to fail
+		or
+			has "midas: proven worthy"
+			not "outfit: Gold Bars"
+
+mission "Midas: Nemo Me Impune Lacessit 2"
+	minor
+	landing
+	invisible
+	repeat
+	source
+		near Sol 100
+	to offer
+		"outfit: Gold Bars" > 15
+		not "midas: sold bars"
+		not "midas: proven worthy"
+		random < 1
+	npc disable
+		government "Midas"
+		personality plunders disables heroic nemesis marked
+		system
+			distance 2 5
+		fleet "Marauders" 5
+		dialog phrase "midas defeated"
+	to fail
+		or
+			has "midas: proven worthy"
+			not "outfit: Gold Bars"
+
+
+
+# SPOILER: Midas sometimes saves pirates from slavery, in exchange for a life debt.
+# Some of your crew may be part of that, or maybe it is just any pirate from this world that tries to get them, attracted by the huge reward.
+mission "Midas: Betrayal For The Bars"
+	landing
+	invisible
+	source
+		government "Pirate"
+	to offer
+		not "midas: sold bars"
+		or
+			random < 40
+			has "Midas: Who is the mad one?: active"
+		has "<uhai> 3: Jump Drive Vault: done"
+		"flagship crew" > 4
+		"outfit: Gold Bars" >= 5
+	on offer
+		conversation
+			`As you land on the pirate world, you head for the hatch, preparing to enter the spaceport, but passing by the cargo bay you think you hear a sound. Odd... There shouldn't be anyone there right now, knowing your crew's habits, as most of them have already disembarked.`
+			choice
+				`	(It's nothing. Probably just a rat.)`
+				`	(Go investigate.)`
+					goto investigate
+			action
+				outfit "Gold Bars" -5
+			`	You decide not to check it out now. A few moments later, you hear a big metallic sound, followed by deep silence. You may want to check your cargo bay later on.`
+				defer
+			label investigate
+			`	You pull out your sidearm and approach the bay as discreetly as you can. Worst case scenario is you find one of your own crew, in which case you better not overreact, or someone could get hurt.`
```

Comment:
```suggestion
			`	You pull out your sidearm and approach the bay as discreetly as you can. Worst case scenario is you find one of your own crew, in which case you had better not overreact, or someone could get hurt.`
```

Reply:
@Hurleveur what about this one?

Reply:
I just have it done in local and I was waiting for more stuff  before committing it
I wasnt sure if this was better which is why I didnt push it with the rest, but after looking online you're right

Reply:
Good point.